app/views/overview/_list-row.html

```diff
@@ -18,4 +18,10 @@
     <bind-service target="row.overlay.state.target"
                   on-close="row.closeOverlayPanel"></bind-service>
   </div>
+  <div ng-if="row.overlay.panelName === 'unbindService'">
+    <unbind-service
+      target="row.overlay.state.target"
+      bindings-by-instance="row.bindings"
```

Comment:
Ah, so it's not a map. I'd call the attribute bindings if it's just the array of bindings. Sorry if I misunderstood.